lib/src/client/call.dart

```diff
@@ -80,6 +80,55 @@ class CallOptions {
   }
 }
 
+/// Runtime options for gRPC-web.
+class WebCallOptions extends CallOptions {
+  final bool bypassCorsPreflight;
```

Comment:
Consider adding doc comment for there two parameters to explain what they do as they are part of public interface, but not self explanatory (especially the CORS bypass which requires correct proxy support)